Packs/Pwned/Integrations/PwnedV2/PwnedV2.py

```diff
@@ -64,7 +67,7 @@ def http_request(method, url_suffix, params=None, data=None):
             wait_amount = wait_regex.group()
         else:
             demisto.error('failed extracting wait time will use default (5). Res body: {}'.format(res.text))
-            wait_amount = 5
+            wait_amount = '5'
```

Comment:
Why this change?

Reply:
lint errors

Reply:
Please revert this. And check why the lint fails. Maybe the declaration was wrong from the beginning.
There is a difference between string and int - timeout seems to be an int to me

Reply:
What about this? are you sure about this change?